lib/lit/cloud.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Lit
+  module Cloud
+    # Sets the provider for cloud translations.
+    # @param [Class] provider Selected translation provider,
+    #   descending from Lit::Cloud::Providers::Base
+    module_function def provider=(provider)
```

Comment:
`module_function` can be used as `private` - once to set a "section" - it might just be that prepending every method in this module is overly explicit